codecov.yml

```diff
@@ -10,14 +10,14 @@ comment:
   show_carryforward_flags: true
 
 ignore:
-  - "bentoml/testing/**/*"
+  - "bentoml/testing"
+  - "bentoml/types.py"
   - "bentoml/_internal/types.py"
-  - "bentoml/_internal/external_typing/**/*"
+  - "bentoml/_internal/external_typing"
```

Comment:
why make this change? was it not working before? could you link to related docs on codecov side?

Reply:
it actually never works before. https://docs.codecov.com/docs/ignoring-paths

Reply:
`Please use this folder/**/*, which will exclude all files within the given folder.`

Reply:
Yet somehow :(
 
![Screenshot 2022-06-10 at 08 45 13](https://user-images.githubusercontent.com/29749331/173066992-e55ee709-3928-4112-ad31-1074845d55ff.png)